roles/kubernetes-apps/helm/tasks/main.yml

```diff
@@ -76,7 +76,6 @@
     | {{ bin_dir }}/kubectl apply -f -
   changed_when: false
   when:
-    - (tiller_override is defined and tiller_override) or (kube_version is version('v1.11.1', '>='))
```

Comment:
this is an `or` - so i   think we must keep `(tiller_override is defined and tiller_override)` here

Reply:
if `kube_version >= v1.11.1`, `(tiller_override is defined and tiller_override) or (kube_version is version('v1.11.1', '>='))` is always true，even `tiller_override is not defined`, we also should override spec.template.spec.priorityClassName .

Reply:
Ah ofc. You are right!